yjit/src/codegen.rs

```diff
@@ -5432,7 +5422,7 @@ fn gen_leave(
     asm.comment("pop stack frame");
     let incr_cfp = asm.add(CFP, RUBY_SIZEOF_CONTROL_FRAME.into());
     asm.mov(CFP, incr_cfp);
-    asm.mov(Opnd::mem(64, EC, RUBY_OFFSET_EC_CFP), incr_cfp);
+    asm.mov(Opnd::mem(64, EC, RUBY_OFFSET_EC_CFP), CFP);
```

Comment:
This is to eliminate a dependency and make an eventual peephole optimization easier